2-mandatory/1-weather-report.js

```diff
@@ -11,8 +11,15 @@
         - Hint: you can call the temperatureService function from your function
 */
 
-function getTemperatureReport(cities) {
-    // TODO
+
+function getTemperatureReport(cities ) {
+    const statements = [];
+    for ( const city of cities){
+    let degrees = temperatureService (city);
+    let statement = ("The temperature in " + city + " is " + degrees + " degrees");
+    statements.push(statement);
+}
+return statements;
 }
 
```

Comment:
Hi Afsoon 
I sew you done nice work here with big effort and I want add here you can use .forEach method here kind of short cut with open big chances to know how function it work and make it more fun easy.
check [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) for more info

this is my review for this week 

Thank Afsoon